README.md

```diff
@@ -235,11 +235,11 @@ Then, add these lines to the composer.json of the **Laravel skeleton application
     }
 ```
 
-Afterwards, install the Aimeos shop package using
+Afterward, install the Aimeos shop package using
 
-`php composer update -W`
+`composer update -W`
```

Comment:
This is related to `wget https://getcomposer.org/download/latest-stable/composer.phar -O composer` and without `php`, it won't work